fork.yaml

```
@@ -46,6 +56,9 @@ def:
             - "params/config.go"
             - "params/protocol_params.go"
             - "core/genesis.go"
+        - title: "E2E tests"
+          globs:
+            - "e2e_test/*"
```

It's the old problem of not having `**` again, so we need
```suggestion
            - "e2e_test/*"
            - "e2e_test/*/*"
```